index.html

```diff
@@ -100,229 +244,79 @@ <h1>I'm Nasir Glad to see you!</h1>
-                <div class="skills-data">
-                  <h3>Skills</h3>
-                </div>
-                <div class="dropdown-icon">
-                  <i class="fa-solid fa-caret-down"></i>
-                </div>
+              <div class="dropdown-icon">
+                <i class="fa-solid fa-caret-down"></i>
+              </div>
+            </div>
+            <div class="about-dropdown">
+              <div class="framworks-data">
+                <h3>Frameworks</h3>
+              </div>
+              <div class="dropdown-icon">
+                <i class="fa-solid fa-caret-down"></i>
+              </div>
+            </div>
+            <div class="about-dropdown">
+              <div class="skills-data">
+                <h3>Skills</h3>
+              </div>
+              <div class="dropdown-icon">
+                <i class="fa-solid fa-caret-down"></i>
               </div>
             </div>
           </div>
         </div>
-      </section>
-      <!-- End About section -->
-      <!-- Start contact section -->
-      <section id="contact">
-        <div class="container">
+      </div>
+    </section>
+    <!-- End About section -->
+    <!-- Start contact section -->
+    <section id="contact" class="flex-row justify-center align-center">
+      <div class="container grid column">
+        <div>
           <h2>Contact me</h2>
           <p>
             If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it!
+            feature that you need built or a project that needs coding. I’d
+            love to help with it!
           </p>
-          <form action="#">
-            <input type="text" placeholder="full name" class="contact-input" />
-            <input
-              type="text"
-              placeholder="@example.gmail.com"
-              class="contact-input"
-            />
-            <textarea
-              name=""
-              id=""
-              cols="0"
-              rows="6"
-              placeholder="your message here..."
-              class="contact-input"
-            ></textarea>
-            <input type="button" class="btn" value="Get in touch" />
-          </form>
         </div>
-      </section>
-      <!-- End contact section -->
-    </main>
-    <!-- End main section -->
-  </body>
-  <!-- End HTML document -->
-</html>
+        <form action="https://formspree.io/f/xrgvogvr" method="POST" class="flex-column align-center">
+          <label for="name">Full name</label>
+          <input type="text" maxlength="30" name="name" placeholder="&nbsp;&nbsp;Full name" class="contact-input"
+            required />
+          <label for="email">Email Address</label>
+          <input type="email" name="email" placeholder="&nbsp;&nbsp;Example@gmail.com" class="contact-input" required />
+          <label for="message">Your message</label>
```

Comment:
Kindly consider removing the label tags since it's not needed in the design template. Instead, you can use the `aria-label` attribute on your input. `aria-label` can be used in cases where text that could label the element is not visible, The purpose of `aria-label` is to provide an accessible name for an element.  Click [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to learn more